### ISSUE TITLE
hide inactive panels from screen reader / tab order

### ISSUE THREAD

**Issue by Lane** [Hyperobjekt/untd-map#70]
it is currently possible to tab into a hidden panel.  content in inactive panels should be hidden from screen readers and keyboard navigation.
